### PR TITLE
Doma-3858 commission calculation

### PR DESCRIPTION
**Pull request by dkoviazin**
1. Added ability to set max and min commission 
2. Added ability to set commission for certain billing category
3. Renamed payment category for trash service and turned it on

### CODE REVIEW

File: apps/condo/domains/acquiring/gql.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const AcquiringIntegration = generateGqlQueries('AcquiringIntegration', ACQUIRIN
 const ACQUIRING_INTEGRATION_ACCESS_RIGHT_FIELDS = `{ user { id } integration { id } ${COMMON_FIELDS} }`
 const AcquiringIntegrationAccessRight = generateGqlQueries('AcquiringIntegrationAccessRight', ACQUIRING_INTEGRATION_ACCESS_RIGHT_FIELDS)
 
-const ACQUIRING_INTEGRATION_CONTEXT_FIELDS = `{ integration { id name explicitFeeDistributionSchema { recipient percent } } organization { id } state settings ${COMMON_FIELDS} implicitFeeDistributionSchema { recipient percent } email reason recipient { bic bankAccount iec tin } }`
+const ACQUIRING_INTEGRATION_CONTEXT_FIELDS = `{ integration { id name explicitFeeDistributionSchema { recipient percent min max category } } organization { id } state settings ${COMMON_FIELDS} implicitFeeDistributionSchema { recipient percent min max category } email reason recipient { bic bankAccount iec tin } }`
```
Comment (SavelevMatthew) [inline apps/condo/domains/acquiring/gql.js:18]: abstract `min` and `max` are not clear to read...Is it `minFee` or `minPercent`

File: apps/condo/domains/acquiring/utils/serverSchema/feeDistribution.js [hunk 2]
```diff
@@ -8,17 +8,58 @@
...
 */
 
-/*  Fee settings
+/*
+    Differences between service fee and commission
+    Commission:
+        1. Is calculated based on the billingReceipt amount. For $1000 with commission 1.2% - commission will be $12
+        2. Can have minimum and maximum amount
+    Service fee:
+        1. Is calculated based on formula ↑. Because, acquiring get commission based on total payment (billingReceipt + serviceFee)
```
Comment (AntonAL) [inline apps/condo/domains/acquiring/utils/serverSchema/feeDistribution.js:17]: It's hard to understand
Could you please explain calculation in order.
Like
Amount = 
Comission =
Service fee =
Acquiring fee =
Additional amount =

File: apps/condo/domains/acquiring/utils/serverSchema/feeDistribution.js [hunk 3]
```diff
@@ -8,17 +8,58 @@
...
+        For $1000 with serviceFee 0.4% and acquiring fee 0.8% - additional amount will be (0.008 + 0.004) ⋅ $1000 / (1 - 0.008) = $12.10
```
Comment (AntonAL) [inline apps/condo/domains/acquiring/utils/serverSchema/feeDistribution.js:18]: What is `(1 - 0.008)`?

```suggestion
        For $1000 with serviceFee 0.4% and acquiring fee 0.8% - additional amount will be (0.004 + 0.008) ⋅ $1000 / (1 - 0.008) = $12.10
```

Comment (dkoviazin) [reply apps/condo/domains/acquiring/utils/serverSchema/feeDistribution.js:18]: The formula is on the top of file

File: apps/condo/domains/acquiring/gql.js (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/domains/acquiring/gql.js]: Done

**Review by SavelevMatthew — APPROVED** [review]

**Pull request merged**